bin/ttt

```diff
@@ -3,19 +3,5 @@ lib = File.expand_path("../../lib", __FILE__)
 $:.unshift(lib)
 
 require 'tictactoe'
-require 'game_io'
-require 'wrapper_stdio'
-require 'player_human'
-require 'player_ai_basic'
-require 'player_ai_minimax'
-require 'game_rules'
 
-io = GameIo.new(WrapperStdio.new)
-
-p1 = PlayerHuman.new('X', io)
-
-p2 = PlayerAiMinimax.new('O', GameRules.new)
-
-
-myGame = TicTacToe.new(io, p2, p1)
-myGame.play_game
+myGame = TicTacToe.new
```

Comment:
If the `bin/ttt` executable remains in the gem, how would clients of the gem use it?